languages/ruby/config.json

```
@@ -10,6 +10,12 @@
   },
   "exercises": {
     "concept": [
+      {
+        "slug": "arrays",
+        "uuid": "4d271980-ab4b-11ea-bb37-0242ac130002",
+        "concepts": ["arrays", "each-loops"],
+        "prerequisites": ["classes", "booleans", "conditionals"]
```

If I look at the example implementation, it doesn't use `each`. Instead, it uses blocks. This makes total sense to me. As blocks are quite an involved concept, I think the best way to go about this is to remove the `each-loops` concept from this exercise and add blocks as a prerequisite.

```suggestion
        "concepts": ["arrays"],
        "prerequisites": ["classes", "booleans", "conditionals", "blocks"]
```

---

I agree. And also remove the bit of text about it in the introduction/after.

Once this is done, let's get this merged. Thank you!